presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -820,6 +859,35 @@ public StageExecutionDescriptor getStageExecutionDescriptor()
         }
     }
 
+    public abstract static class CustomVisitor
```

Comment:
small naming thing.  this isn't actually a visitor (doesn't implement visitor pattern).  I think that's okay (the CustomXXX could call an internal visitor if it needs to go through the tree) , but we should call it something else e.g. customPlanTranslator or something.